test/lev-top-nav/Index.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import 'jest-styled-components';
+import LevTopNav from "../../src/lev-top-nav/index";
+import renderer from 'react-test-renderer';
+
+describe('component', () => {
+  let tree, root;
+  beforeEach(() => {
+    tree = renderer.create(<LevTopNav />);
+    root = tree.root;
+  });
+
+  it('renders correctly', () => expect(tree.toJSON()).toMatchSnapshot());
+
+  it('contains 3 links with page-link class', () => {
+    let links = root.findAllByProps({className: "page-link"});
+    expect(links.length).toBe(3);
+  })
+
+  it('contains a report link', () => {
+    let link = root.findByProps({className: "page-link", href: '/'});
+    expect(link.props.children).toEqual('Report');
```

Comment:
This is no barrier to merging but...

This could probably be made less brittle by simply searching for the presence of the string "Report" in the text representation. (Assuming the API lets you do that.)

Reply:
As discussed - accessing text nodes with the API is, at best, a PITA...left as is.